app/assets/stylesheets/classified_listings.scss

```diff
@@ -381,12 +381,44 @@
 .listingform__tagoptionrow {
   padding: 10px;
   cursor: pointer;
+  position: relative;
   &:hover {
     background: lighten($green, 27%);
     color: $black;
   }
 }
 
+.listingform__tagsoptionsbottomrow {
+  display: none; //not yet fully functional or relevant
```

Comment:
Same question as below: Should we commit code that doesn't work yet?

Reply:
I made an adjustment to remove the non-used styles. I'm going to keep this approach as a de facto feature flag when weighed against the additional code it might take to adjust the abstracted `tags` component.

I think this is a nuanced discussion but this is the cleanest practical solution for now IMO.